lib/Loop/Loop.php

```diff
@@ -88,7 +88,7 @@ public function setInterval(callable $cb, float $timeout): array
     /**
      * Stops a running interval.
      *
-     * @param array<int, mixed> $intervalId
+     * @param array{0:string, 1:boolean} $intervalId
```

Comment:
Is there a way to declare that the boolean is a reference to a boolean?

Reply:
not that I am aware of.

but maybe it makes sense to use a [`phpstan-type`](https://phpstan.org/writing-php-code/phpdoc-types#local-type-aliases) so one can see that we should pass something in here, which was returned by `setInterval`